.devcontainer/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:22.04
+
+# Install python
+RUN apt-get update
+RUN apt-get install -y \
+    python3 \
+    python3-venv \
+    python3-pip
+
+# Dev tools require an additional update
+RUN apt-get update
+RUN apt-get install -y \
+    git \
+    gdb \
+    curl \
+    zip \
+    unzip \
+    tar \
+    autoconf \
+    libtool \
+    libtool-bin \
+    pkg-config
```

Comment:
It's not an issue, but I believe the build-essential metapackage includes git, gdb, gcc, autconf, libtool, automake, and pkgconfig. It's sometimes handy if you need to set up the environment without needing to remember all the individual packages.

I don't know if you also need automake. Presumably not if the builds are running.

Reply:
For the GitHub CI images like ubuntu-latest these tools are present, but for the regular container I had to add them manually. It seems that the default ubuntu image is very minimal (no git for example).